frame/ddc-staking/src/testing_utils.rs

```diff
@@ -0,0 +1,155 @@
+//! Testing utils for ddc-staking. 
+
+use crate::{Pallet as DDCStaking, *};
```

Comment:
Please check if it could be `DdcStaking` by naming convention.

Reply:
Ok will rename